test/integration/connect/envoy/case-cfg-splitter-features/verify.bats

```diff
@@ -79,7 +79,7 @@ load helpers
 
 @test "test response header manipulation" {
   # Add a response header that should be stripped by the route.
-  run retry_default curl -v -f -X PUT \
+  run retry_default curl -m 5 -v -f -X PUT \
```

Comment:
This setting will be overriden by
`if [[ $1 == "curl" ]]; then
    set -- ${@} -m 5
  fi`
So I think maybe there is no point in setting this timeout here in the verify.bats file.

Reply:
Yes, that check was implemented. I probably forgot to remove that parameter from here